molotov/util.py

```diff
@@ -121,3 +124,45 @@ def expand_options(config, scenario, args):
         raise OptionError("Can't find %r in the config" % scenario)
 
     _expand_args(args, config['molotov']['tests'][scenario])
+
+
+def _run_in_fresh_loop(coro):
+    loop = asyncio.new_event_loop()
+    res = None
+    try:
+        task = loop.create_task(coro(loop=loop))
+        res = loop.run_until_complete(task)
+    finally:
+        loop.close()
```

Comment:
looks like this allows the loop to be closed cleanly (and prevents the errors we were seeing before)